qt/python/mantidqt/test/test_algorithm_dialog.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from qtpy.QtWidgets import QWidget, QLineEdit
+
+from mantid.api import PythonAlgorithm
+from mantid.kernel import Direction, FloatArrayProperty
+from mantid.simpleapi import AlgorithmManager, AlgorithmFactory
```

Comment:
These classes can also be imported from `mantid.api`. I'd like to keep the`simpleapi` as a high-level module for users.